modules/ROOT/pages/tutorial-services.adoc

```diff
@@ -15,7 +15,7 @@ Let's write a simple script that uses https://icanhazip.com/[icanhazip.com] to u
 ----
```

Comment:
> Let's write a simple script that uses https://icanhazip.com/[icanhazip.com] to update the issuegen from https://github.com/coreos/console-login-helper-messages[console-login-helper-messages] to output the node's public IPv4 address on the serial console during bootup:

I think we can omit the mention of CLHM here since it's not really involved and we are no longer modifying the issuegen created by CLHM

Reply:
Perhaps
> Let's write a simple script that uses icanhazip.com to update the `issue` file (prelogin message and identification file)...